src/System.Data.SqlClient/tests/FunctionalTests/ExceptionTest.cs

```diff
@@ -98,6 +99,7 @@ public void NamedPipeTest(string dataSource)
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)] // Different exception messages in Framework
```

Comment:
If the only thing that differs is the exception message, would it not be better to just special-case that check inside the test, so the rest of it still executes?  That'll help to verify the behavior on .NET Framework and thus help to serve as an oracle for the .NET Core behavior matching against it.